falkordb/falkordb.py

```diff
@@ -225,6 +227,40 @@ def config_get(self, name: str) -> Union[int, str]:
 
         return self.connection.execute_command(CONFIG_CMD, "GET", name)[1]
 
+
+    def get_replica_connections(self):
+        """
+        Retrieve a list of connections for FalkorDB replicas.
+
+        This function determines the FalkorDB setup (Sentinel or Cluster) and returns 
+        the hostnames and ports of the replicas.
+
+        Returns:
+            list of tuple: A list of (hostname, port) tuples representing the 
+            replica connections.
+
+        Raises:
+            ConnectionError: If unable to connect or retrieve information from 
+            the FalkorDB setup.
+            ValueError: If the `mode` is neither Sentinel nor Cluster.
+        """
+        # decide if it's Sentinel or cluster
+        try:
+            mode = self.connection.execute_command("info")['redis_mode']
+        except Exception as e:
+            raise Exception(f"Failed to get Redis mode: {e}")
```

Comment:
if `hasattr(self, 'sentinel')` is true, then performing this action is a waste of time.